clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -197,7 +199,15 @@ export class CopybookDownloadService implements vscode.Disposable {
             return;
         }
         if (CopybookDownloadService.isEligibleForCopybookDownload() && !ZoweVsCodeExtension.getZoweExplorerApi()) {
-            if (!quiet) { vscode.window.showErrorMessage(ZOWE_EXT_MISSING_MSG); }
+            if (!quiet) {
+                vscode.window.showErrorMessage(ZOWE_EXT_MISSING_MSG, INSTALL_ZOWE)
+                .then(action => {
+                    if (action === INSTALL_ZOWE) {
+                        vscode.commands.executeCommand("workbench.extensions.installExtension", "zowe.vscode-extension-for-zowe");
+                        vscode.commands.executeCommand("workbench.extensions.search", "zowe.vscode-extension-for-zowe");
```

Comment:
Why do you call both commands? I think that it will be OK to call only install or search.

Reply:
a search would be ok I guess. let the user install by himself.